packs/spell-effects/spell-effect-life-link.json

```diff
@@ -0,0 +1,87 @@
+{
+    "_id": "8SkrekdJ9h3T664t",
+    "img": "systems/pf2e/icons/spells/life-link.webp",
+    "name": "Spell Effect: Life Link",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Life Link]</p>\n<p>You forge a connection of life energy between you and your target, distributing harm between both of you.<br /><br /><strong>Apply Damage (Caster)</strong> @Damage[(3)[Hit Points]].</p>\n<p><strong>Heightened (3rd)</strong> @Damage[(5)[Hit Points]].</p>\n<p><strong>Heightened (6th)</strong>@Damage[(10)[Hit Points]].</p>\n<p><strong>Heightened (9th)</strong>@Damage[(15)[Hit Points]].</p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "minutes",
+            "value": 1
+        },
+        "level": {
+            "value": 1
+        },
+        "publication": {
+            "license": "OGL",
+            "remaster": false,
+            "title": "Pathfinder Advanced Player's Guide"
+        },
+        "rules": [
+            {
+                "alwaysActive": false,
+                "domain": "all",
+                "key": "RollOption",
+                "label": "Life Link",
+                "option": "life-linked",
+                "predicate": [
+                    {
+                        "not": "self:signature:{item|origin.signature}"
+                    }
+                ],
+                "toggleable": true
+            },
+            {
+                "key": "ActiveEffectLike",
+                "mode": "override",
+                "path": "system.attributes.hardness.value",
```

Comment:
Sorry, but this isn't a method we will use to implement this. Life link isn't hardness.

Reply:
Fair enough - Do you want me to close this PR then?